README.md

```diff
@@ -69,22 +69,26 @@ Create an example template to easily structure the lochness system
 # ProNET
 lochness_create_template.py \
     --outdir /data/lochness_root \
-    --studies BWH McLean \
+    --studies PronetLA PronetSL PronetWU \
     --sources redcap xnat box mindlamp \
     --email kevincho@bwh.harvard.edu \
     --poll_interval 43200 \
     --ssh_host erisone.partners.org \
-    --ssh_user kc244
+    --ssh_user kc244 \
+    --lochness_sync_send \
+    --s3
 
 # PRESCIENT
 lochness_create_template.py \
     --outdir /data/lochness_root \
-    --studies BWH McLean \
-    --sources RPMS daris mediaflux mindlamp \
+    --studies PrescientAD PrescientME PrescientPE \
+    --sources RPMS mediaflux mindlamp \
     --email kevincho@bwh.harvard.edu \
     --poll_interval 43200 \
     --ssh_host erisone.partners.org \
```

Comment:
The purpose of `ssh_host` and `ssh_user` was lochness to lochness sync through a VM. Here since you are attempting the same sync through `--s3`, shouldn't the former two arguments be omitted? If `sync.py` is wrote to require them, can we make them optional?

Reply:
No the purpose of `ssh_host` and `ssh_user` was for the email functionality built-in in the original `lochness`. Please feel free to make it optional.